src/ui/room_properties.ui

```diff
@@ -127,10 +127,19 @@
       <string>Symbol</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_groupBox_symbol">
+      <property name="sizeConstraint">
+       <enum>QLayout::SetMinimumSize</enum>
+      </property>
       <item>
        <widget class="QLabel" name="label_symbolInstructions">
+        <property name="sizePolicy">
+         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
+          <horstretch>0</horstretch>
+          <verstretch>0</verstretch>
+         </sizepolicy>
+        </property>
         <property name="text">
-         <string>&lt;p&gt;Choose: &lt;br/&gt;• an existing symbol from the list below (sorted by most commonly used first)&lt;br/&gt;• enter one or more graphemes (&amp;quot;visible characters&amp;quot;) as a new symbol&lt;br/&gt;• enter a space to clear any existing symbols&lt;br/&gt;for all of the %n selected room(s):&lt;/p&gt;</string>
+         <string notr="true">&lt;p&gt;Dummy text replaced when used.&lt;/p&gt;</string>
```

Comment:
Note that the `notr="true"` means that these text will not be fed into the translation system - so will NOT result in more work for the translators. :grinning: